deepspeed/module_inject/replace_policy.py

```diff
@@ -405,8 +405,8 @@ class MegatronLayerPolicy(TransformerPolicy):
     version = 0
     moe_type = 'standard'
 
```

Comment:
Let's add a megatron_v2 argument here, so that the user can control this if needed to turn it off and use the previous layout.

Reply:
Updated in latest commit, thanks for the feedback